### DISCUSSION TITLE
FUSE mount gRPC - fixed port?

### DISCUSSION THREAD

**Discussion by ghost** [seaweedfs/seaweedfs#2187] [Q&A]
When using `weed mount -filer=ip:portA`, we can specify the HTTP port to be used.

`weed` assumes the gRPC port is `portA + 10000`. This might not be the case if (for example) you are running the filer with its ports forwarded to dynamic ports (e.g. nomad, kubernetes..)

Is there a way around this that I'm not seeing (without changing my setup)? If not, maybe it should be possible to specify the gRPC port like the HTTP port..

**Comment by chrislusf** [comment]
Currently only `grpcPort = httpPort + 10000`. So instead of one customizable port, the httpPort and grpcPort are also "customizable", but only in pair.

When a customizable grpcPort is needed in nomad/k8s?

**Comment by ghost** [comment]
Just saw [the notes about this in the 2.68 release](https://github.com/chrislusf/seaweedfs/releases/tag/2.68) - will give it a try soon, thanks!
